bundles/org.openhab.binding.remoteopenhab/src/main/java/org/openhab/binding/remoteopenhab/internal/rest/RemoteopenhabRestClient.java

```diff
@@ -400,17 +400,15 @@ private void onEvent(InboundSseEvent inboundEvent) {
             connected = true;
             listeners.forEach(listener -> listener.onConnected());
         }
+        if (!"message".equals(name)) {
+            if (!"alive".equals(name)) {
```

Comment:
Any reason you can't silently ignore anything other than "message"?  Seems like SSE types the client doesn't know about does not want to react too should just be silently discarded.

Reply:
It was to detect potential new messages types not yet supported by the binding. 
But yes, I could rather ignore them silently.

Reply:
Done